src/.nuget/Microsoft.NETCore.Platforms/readme.txt.txt

```diff
@@ -0,0 +1 @@
+TODO: delete this file and pull it from either MS.NETCore.Platforms package or fix buildtools to carry a copy.
```

Comment:
To address this just add a project.json to the repo (any where that'll be restored) that references Microsoft.NETCore.Platforms, then you can change the property to use the path to the file in the restored packages directory.